packages/dds/tree/src/simple-tree/api/schemaCompatibilityTester.ts

```diff
@@ -187,19 +149,9 @@ export class SchemaCompatibilityTester {
 							canUpgrade = false;
 						}
 					} else {
-						// Node type exists in both schemas but has changed. We conservatively never allow this.
-						const storedIsNever =
-							storedNodeSchema !== undefined
-								? isNeverTree(this.policy, stored, storedNodeSchema)
-								: true;
-						const viewIsNever =
-							viewNodeSchema !== undefined
-								? isNeverTree(this.policy, this.viewSchemaAsStored, viewNodeSchema)
-								: true;
-						if (!storedIsNever || !viewIsNever) {
-							canView = false;
-							canUpgrade = false;
-						}
+						// Node type exists in both schemas but kind has changed. We conservatively never allow this.
```

Comment:
Maybe merge this comment and the one above on this subject (possible moving that one down here. Or do something like:

```suggestion
						// Node type exists in both schemas but kind has changed. We conservatively never allow this.
						// See note above about cases where this could be allowed if needed.
```